package.json

```diff
@@ -21,21 +21,17 @@
   },
   "devDependencies": {
     "babel-cli": "^6.9.0",
-    "babel-plugin-transform-object-rest-spread": "^6.8.0",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "create-react-class": "^15.5.3",
-    "happiness": "^6.0.7",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "happiness": "^10.0.2",
     "mocha": "^2.5.3",
-    "mochify": "^2.18.0",
-    "prop-types": "^15.5.10",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4"
+    "mochify": "^2.18.0"
   },
   "peerDependencies": {
-    "react": ">0.12.0"
-  },
-  "dependencies": {
-    "@streammedev/react-compat": "0.0.0"
+    "react": "^16.0.0",
```

Comment:
Change these to just `>15.0.0`.  I think anything above that should support the `React.Component` class extension.